### ISSUE TITLE
Mythic Metals load crash

### ISSUE THREAD

**Issue by xg787** [Sinytra/Connector#1339]
### Mod Name

Mythic Metals

### Mod Homepage

https://modrinth.com/mod/mythicmetals

### Minecraft version

1.21.1

### Describe the bug

crash on startup

### Steps to reproduce

1.load mythic metals and dependencies
2.crash

### Logs

https://gist.github.com/xg787/93564eb0a2d7960baa7e5098677ef8ee

**Comment by Astral3** [comment]
Same problem on 1.20.1

**Comment by Tapacywka** [comment]
Works for me on 1.20.1
![image](https://github.com/user-attachments/assets/083042f6-bd47-48c4-892d-a361b6e29d21)
![image](https://github.com/user-attachments/assets/bab4b5b9-32f5-4d89-89b2-c58b8c656cac)
![image](https://github.com/user-attachments/assets/95ed90aa-8ef8-4856-9363-795fd810cbd8)

**Comment by Astral3** [comment]
Strange. I'll try it again.

**Comment by Astral3** [comment]
> Works for me on 1.20.1
> Strange. I'll try it again.


No, this still occurs for me. Its possible it happens due to some sort of other mod compatibility issue, but I can't diagnose which one it would be without using the binary method. None of the logs report the error, and no crash report is generated - the instance simply terminates while in early loading. I simply assumed it must be the connector, since a report already existed. 

Sorry 'bout that.

**Comment by Xplodin** [comment]
seems like alloy forgery is why its crashing. No idea why (1.21.1)
